hack/bicep-types-radius/generated/applications/applications.core/2023-10-01-preview/types.json

```diff
@@ -1958,6 +2001,14 @@
       "$ref": "#/0"
     }
   },
+  {
+    "$type": "ObjectType",
+    "name": "RecipeConfigPropertiesEnvSecrets",
+    "properties": {},
+    "additionalProperties": {
+      "$ref": "#/149"
+    }
+  },
```

Comment:
existing types.json has slightly different format/casing.  checking once that it is expected
```
 "envSecrets": {
          "Type": 158,
          "Flags": 0,
          "Description": "Environment variables containing sensitive information can be stored as secrets. The secrets are stored in Applications.Core/SecretStores resource."
        }
      }